### DISCUSSION TITLE
Commit with `<C-c><C-c>` and abort with `<C-c><C-k>`

### DISCUSSION THREAD

**Discussion by alexmozaidze** [NeogitOrg/neogit#492] [Show and tell]
In Doom Emacs, I was used to `<C-c><C-c>` and `<C-c><C-k>`, and so I made it into a thing here:
```lua
vim.api.nvim_create_autocmd("BufEnter", {
    pattern = { "COMMIT_EDITMSG" },
    callback = function()
      vim.keymap.set({ "n", "i" }, "<c-c><c-c>", "<esc><cmd>wq<cr>", { noremap = true, buffer = true })
      vim.keymap.set({ "n", "i" }, "<c-c><c-k>", "<esc><cmd>q!<cr>", { noremap = true, buffer = true })
    end,
})
```

**Comment by theherk** [comment]
For some reason when I use this I get a popup `Type :qa and press enter to exit nvim`, the second time I get `Type :qa and press enter to abandon changes and exit nvim`, then on the third time it works. I cannot sort out what is going on. Not a big deal, I can just use `:x`, but if you have ideas let me know.
